wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,27 @@
+<form [formGroup]="registerForm" (ngSubmit)="addPost(registerForm.value)" class="form-container" novalidate>
```

Comment:
If you bind the inputs to controller variables, there's no need to pass the form value to `addPost`